charmcraft.yaml

```diff
@@ -12,6 +12,7 @@ bases:
 
 parts:
   charm:
+    # todo: remove binary packages
```

Comment:
noob question, why do these need to be removed? last I checked charm wouldn't pack without these